groups/managers.py

```diff
@@ -1,8 +1,19 @@
+import datetime
+
+from django.apps import apps
 from django.contrib.auth import get_user_model
 from django.db import models
 from polymorphic import PolymorphicManager, PolymorphicQuerySet
 
 
+DEFAULT_ACTIVE_THRESHOLD = apps.get_app_config('groups').default_active_threshold_days
+
+
+def get_threshold_date(threshold_days=DEFAULT_ACTIVE_THRESHOLD):
+    """Return the earliest posting date a comment can have and still be recent."""
+    return datetime.date.today() - datetime.timedelta(days=threshold_days)
```

Comment:
It seems more sensible to me.  The idea I was going for is "within the last N days" and that doesn't generally come with provisos about what time it was.  (At least in my experience, when people want to refer to a specific time N days ago, they'll say N*24 hours.)